testing/conformance/src/vm.rs

```diff
@@ -79,6 +79,12 @@ impl TestMachine<Box<DefaultMachine<MemoryBlockstore, TestExterns>>> {
 
         let machine = DefaultMachine::new(&engine, &mc, blockstore, externs).unwrap();
 
+        // Preload the actors. We don't usually preload actors when testing, so we're going to do
```

Comment:
Due to the way feature unification works, preloading may or may not be enabled by default (because the testing feature may get enabled by some other crate in the build).

So I'm just explicitly enabling the testing feature for conformance tests (also lets us hide types like the StateTree and Gas).

Reply:
Makes sense to me.